docs/ui/auth/fragments/react/authenticator.md

```diff
@@ -326,3 +351,27 @@ const App = () => (
 - </Authenticator>
 );
 ```
+
+If you are using `withAuthenticator`:
+```diff
+- import { withAuthenticator } from 'aws-amplify-react';
++ import { withAuthenticator } from '@aws-amplify/ui-react';
+```
+
+```jsx
+export default withAuthenticator(App);
+```
+
+### Breaking changes for withAuthentciator
+
+<amplify-callout warning>
+
+If you were providing additional options to `withAuthenticator` (e.g. `includeGreetings`, `authenticatorComponents`, `federated`, `theme`), these have changed
```

Comment:
add a period to the sentence.

Reply:
and provide a sentence maybe like: "Review the updated properties [here](#properties)"

Reply:
done